ablog/post.py

```diff
@@ -13,13 +13,22 @@
 from docutils import nodes
 from sphinx.locale import _
 from sphinx.util.nodes import set_source_info
-from sphinx.util.compat import Directive, make_admonition
+from sphinx.util.compat import Directive
+# DeprecationWarning: make_admonition is deprecated, use docutils.parsers.rst.directives.admonitions.BaseAdmonition instead
+from docutils.parsers.rst.directives.admonitions import BaseAdmonition as make_admonition
```

Comment:
Above should be like
```Python
try:
    from sphinx.util.compat import Directive, make_admonition		
except ImportError:
    from docutils.parsers.rst.directives.admonitions import BaseAdmonition as make_admonition
```
so that this still works with older versions of Sphinx.